### ISSUE TITLE
DK2 Support

### ISSUE THREAD

**Issue by slawrence** [benvanik/vr.js#35]
Any plans to support DK2 (SDK 0.4)?

**Comment by loktar00** [comment]
+1 wondering that as well.

**Comment by benvanik** [comment]
Probably not; Chrome killed NPAPI so vr.js doesn't work there any more :( Vlad and Brandon are making awesome progress on WebVR, though, so you should be able to grab their Firefox/Chromium builds and play around with that.

There's still the possibility of porting the javascript driver to the chrome.hid, though I probably won't get around to that.

**Comment by loktar00** [comment]
Ah that sucks, I assumed as much though, webvr is quite promising. Regardless, thanks for the work on the vr.js version.

**Comment by captainkw** [comment]
Thanks for the work on the vr.js as well, it made a lot of things really easy for us as well.  Will follow up on Firefox/Chromium builds and hopefully their WebVR format gets adopted into the main trunk soon!

**Comment by benvanik** [comment]
Good luck! Excited to see a lot more fun web VR! I'll try to update the
vr.js main page tonight for others also looking for the same info.

**Comment by CodingCooky** [comment]
see a branch from NPVR for DK2 SDK0.2.4 support:
CodingCooky/npvr

**Comment by captainkw** [comment]
This is good stuff @CodingCooky, will give it a try.  But by any means did you mean SDK 0.4.2 instead of 0.2.4?

**Comment by CodingCooky** [comment]
@captainkw: 
You are totally correct. I was meaning SDK 0.4.2. Thanks for the info - i updated the comments on the other projects.
